cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLTransformationPushdown.java

```diff
@@ -22,19 +22,19 @@
  * Class used to represent the configuration properties that are needed to configure Transformation Pushdown.
  */
 public class ETLTransformationPushdown {
-  private final ETLPlugin engine;
+  private final ETLPlugin plugin;
```

Comment:
Made this change to make this compatible with the UI spec for plugins, hopefully reducing UI implementation effort for the SQL Engine config dialog.